### PR TITLE
Fix clightning prometheus plugin msat purge error

### LINKED ISSUE #616: prometheus clightning plugin error
I am still getting the following Error with nix-bitcoin 0.0.93:

```
lightningd[244296]: UNUSUAL plugin-prometheus.py: Traceback (most recent call last):
lightningd[244296]: UNUSUAL plugin-prometheus.py:   File \"/nix/store/rpri9nb8xpwwqakyrqbg8zdslkjs2hd3-python3-3.10.11/lib/python3.10/wsgiref/handlers.py\", line 137, in run
lightningd[244296]: UNUSUAL plugin-prometheus.py:     self.result = application(self.environ, self.start_response)
lightningd[244296]: UNUSUAL plugin-prometheus.py:   File \"/nix/store/a7c858sc2i95d533qgls5cj6x4avlhy6-python3-3.10.11-env/lib/python3.10/site-packages/prometheus_client/exposition.py\", line 129, in prometheus_app
lightningd[244296]: UNUSUAL plugin-prometheus.py:     status, headers, output = _bake_output(registry, accept_header, accept_encoding_header, params, disable_compression)
lightningd[244296]: UNUSUAL plugin-prometheus.py:   File \"/nix/store/a7c858sc2i95d533qgls5cj6x4avlhy6-python3-3.10.11-env/lib/python3.10/site-packages/prometheus_client/exposition.py\", line 105, in _bake_output
lightningd[244296]: UNUSUAL plugin-prometheus.py:     output = encoder(registry)
lightningd[244296]: UNUSUAL plugin-prometheus.py:   File \"/nix/store/a7c858sc2i95d533qgls5cj6x4avlhy6-python3-3.10.11-env/lib/python3.10/site-packages/prometheus_client/openmetrics/exposition.py\", line 21, in generate_latest
lightningd[244296]: UNUSUAL plugin-prometheus.py:     for metric in registry.collect():
lightningd[244296]: UNUSUAL plugin-prometheus.py:   File \"/nix/store/a7c858sc2i95d533qgls5cj6x4avlhy6-python3-3.10.11-env/lib/python3.10/site-packages/prometheus_client/registry.py\", line 97, in collect
lightningd[244296]: UNUSUAL plugin-prometheus.py:     yield from collector.collect()
lightningd[244296]: UNUSUAL plugin-prometheus.py:   File \"/nix/store/i1rb0jiyfgvggg8cqlc3ir7p6yfqis2k-clightning-plugin-prometheus-eee5e90/prometheus.py\", line 185, in collect
lightningd[244296]: UNUSUAL plugin-prometheus.py:     in_msatoshi_offered_gauge.add_metric(labels, c['in_msatoshi_offered'])
lightningd[244296]: UNUSUAL plugin-prometheus.py: KeyError: 'in_msatoshi_offered'
```
Maybe #613 didn't fully fix the msat purge?
Or this is due to the following although that is a different error? https://github.com/lightningd/plugins/pull/451#issuecomment-1586266746

### PR DESCRIPTION
**Pull request by HaosGames**
Closes #616

https://github.com/lightningd/plugins/pull/451 was merged.

### CODE REVIEW

File: pkgs/clightning-plugins/default.nix [hunk 1]
```diff
@@ -3,25 +3,14 @@ pkgs: nbPython3Packages:
...
-    patches = [
-      # https://github.com/lightningd/plugins/pull/451
-      (pkgs.fetchpatch {
-        name = "202305-prometheus-msat-purge";
-        url = "https://github.com/lightningd/plugins/commit/f8a27b97a1b9ded8790c1f033b1f4268c0a6e210.patch";
-        sha256 = "sha256-0lFMhHHIi9bUU0+xaHhpnascNlFmr51JxE6e2F0s0zc=";
-      })
-    ];
+      rev = "ce078bb74e10b5dea779fcd9fbe77e1d3e72db7a";
+      sha256 = "19gp2wssd0rlbsiq1h5h48lg2q4m6wjyvy2lhmw5vmny6lkx48a8";
```
Comment (erikarvstedt) [inline pkgs/clightning-plugins/default.nix:10]:
```suggestion
      hash = "sha256-SCHSJzXe1l14hVT47SU3lWDxKCKwwICjXjSDpjUX96U=";
```

**Comment by erikarvstedt** [comment]
Thanks! Can you squash the last commit into the previous one?

**Comment by HaosGames** [comment]
Of course. Done

File: pkgs/clightning-plugins/default.nix [hunk 2]
```diff
@@ -3,25 +3,14 @@ pkgs: nbPython3Packages:
 let
   inherit (pkgs) lib;
 
-  src = pkgs.applyPatches {
-    src = pkgs.fetchFromGitHub {
+  src = pkgs.fetchFromGitHub {
       owner = "lightningd";
       repo = "plugins";
```
Comment (erikarvstedt) [inline pkgs/clightning-plugins/default.nix:8]: Can you fix the indentation to use two spaces?
```suggestion
    owner = "lightningd";
    repo = "plugins";
    ...
```

**Review by erikarvstedt — APPROVED** [review]
ACK 8157cd4fe3a475d9c238b046e693331fcb5eb5b8

**Comment by jonasnick** [comment]
Thanks @HaosGames

**Pull request closed without merging**